core/scss/components/date-range-picker/date-range-picker.scss

```diff
@@ -45,4 +45,111 @@
     }
 
   }
+
+  // for angularjs
+  md-date-range-picker.angularjs {
+    display: inline-block;
+
+    .md-date-range-picker {
+      position: relative;
+      display: inline-block;
+      width: 100%;
+
+      &.with-icon {
```

Comment:
I suggest either moving this into NG or adjusting these class names for bems.  As this date range picker goes beyond what our current scss covers @pauljeter